### DISCUSSION TITLE
Character Scanner Directions Confusing

### DISCUSSION THREAD

**Discussion by aelfwyne** [xenesty/AkashaScanner#19] [Q&A]
Trying to run the character scanner, but when I follow the directions as written... exactly. It does not work.

1. Open Character Screen. Okay, I see my 4 characters in my party first, followed by everyone else in level order (across the top). There's no sort option ON THIS SCREEN.
2. Do you mean the character list (four squares button???) Click on it, sort by level as directed. OK.
3. But you say to have the left menu with Attributes, weapons, etc, visible. OK no problemo. Exit back out to that screen from the list (after sorting).
4. Attempt to run the scanner. It just bounces between attributes and constellations on the first character and never does anything.

Can you at least show a picture of what we're supposed to have on our screen to start the scan?

**Comment by xenesty** [comment]
You should start the scanner where you first open the character screen.
